client/state/reader/users/actions.js

```diff
@@ -0,0 +1,57 @@
+import wpcom from 'calypso/lib/wp';
+import {
+	READER_USER_REQUEST,
+	READER_USER_REQUEST_SUCCESS,
+	READER_USER_REQUEST_FAILURE,
+} from '../action-types';
+
+import 'calypso/state/reader/init';
+
+const requestsInFlight = new Set();
+export const requestUser = ( dispatch, ownProps ) => () => {
+	return {
+		requestUser: () => {
```

Comment:
I'm *really* unsure how we should be defining the [`mapDispatchToProps` function](https://react-redux.js.org/api/connect#mapdispatchtoprops-object--dispatch-ownprops--object). I've used `requestUser` here twice because otherwise it doesn't work. I haven't found much good inspiration in other parts of the codebase as of yet.

I might just need to sleep on it, but any suggestions appreciated!